blackjack.py

```diff
@@ -17,12 +17,16 @@ class Card(object):
     def __init__(self, value, suit):
         self.value = value
         self.suit = suit
-        self.points = self.get_card_points()
+        self.points = self.card_points
 
     def __repr__(self):
         return '{} of {}'.format(self.value, self.suit)
 
-    def get_card_points(self):
+    def __str__(self):
+        return '{} of {}'.format(self.value, self.suit)
+
+    @property
+    def card_points(self):
```

Comment:
Why not rename this method to `points` and lose `self.points` in `__init__`?

Reply:
Slack integration: testing testing 1 2 3